src/readers/morphologyHDF5.cpp

```diff
@@ -199,10 +199,10 @@ HighFive::DataSet MorphologyHDF5::_getStructureDataSet(size_t nSections) {
 
 
 /**
-   Returns true if the neuron has no neurites
+   Returns true if the neuron has neurites (for MORPHOLOGY_VERSION_H5_2)
 **/
-static inline bool noNeurites(int firstSectionOffset) {
-    return (firstSectionOffset == -1);
+static inline bool v2HasNeurites(int firstSectionOffset) {
```

Comment:
Any reason for this change. I thought this was not specific to v2.

Reply:
I noticed this function can only used in the context of v2 since in v1, according to the code, if there are no neurites the `firstSectionOffset` is gonna be positive, equal to the full length of the array.

Reply:
Seconded. It's also only used in code paths specific to v2.